packages/promptable/package.json

```diff
@@ -27,6 +27,7 @@
         "csv-parse": "^5.3.4",
         "gpt3-tokenizer": "^1.1.4",
         "openai": "^3.1.0",
+        "ts-toolbelt": "^9.6.0",
```

Comment:
This import is added for `S.Split`. We can probably add split without needing to add this entire library but there are a lot of useful helpers in [`ts-toolbelt`](https://github.com/millsp/ts-toolbelt).